contracts/mocks/ERC4907AMock.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+// ERC721A Contracts v4.1.0
+// Creators: Chiru Labs
+
+pragma solidity ^0.8.4;
+
+import '../extensions/ERC4907A.sol';
+
+contract ERC4907AMock is ERC721A, ERC4907A {
+    constructor(string memory name_, string memory symbol_) ERC721A(name_, symbol_) {}
+
+    function safeMint(address to, uint256 quantity) public {
```

Comment:
I think we should use mint in our mock just so ppl stop trying to use safeMint when they don't need to

Reply:
ok